packages/back-end/test/models/dataSourceModel.test.ts

```diff
@@ -293,7 +294,24 @@ describe("dataSourceModel", () => {
         name: "Updated Data Source",
       };
       await expect(
-        updateDataSource(datasource, "test", updates)
+        updateDataSource(
+          {
+            org: {
+              id: "test",
+              name: "Test",
+              url: "",
+              ownerEmail: "",
+              dateCreated: new Date(),
+              members: [],
+              invites: [],
+              settings: {},
+            },
+            environments: [],
+            readAccessFilter: FULL_ACCESS_PERMISSIONS,
+          },
+          datasource,
+          updates
+        )
```

Comment:
Might want to leave a TODO or make a note to create a test utility for generating a context object .. That way if it changes in the future we have one place to update and lessen chance of bugs slipping through